script.js

```diff
@@ -0,0 +1,253 @@
+
+// Create two variables personWeight and personHeight  and assign some numbers there.
+// Calculate the BMI (and store it in well-named variable) using the formula BMI = weight / (height * height).
+// Print the calculated BMI.
+
+const personWeight = 80
+const personHeight = 1.80
+
+const calculateBMI = personWeight / (personHeight * personHeight);
+
+console.log(calculateBMI);
+
+// Write a program that checks if a person is eligible to vote.
+// Declare a variable called age and assign it a numeric value.
+// Use an if-else statement to determine if the person can vote (age >= 18) and print (using console.log() ) the result:
+// a citizen can vote  if the age is at least 18
+// a citizen can't vote if the age is less than 18
+
+const age = 17;
+
+if (age >= 18) {
+    console.log('A citizen can vote');
+} else if (age < 18) {
+    console.log('A citizen can not vote');
+}
+
+// Write a code that simulates a traffic light.
+// Declare a variable called currentColor with values "red", "yellow", or "green".
+// Use an if statement to print a message based on the current color (e.g., "Stop" for red, "Speed up" for yellow, "Go" for green).
+
+const currentColor = 'yellow';
+
+if (currentColor === ('red')) {
+    console.log('Stop');
+} else if (currentColor === ('yellow')) {
+    console.log('Speed up');
+} else if (currentColor === ('green')) {
+    console.log('Go');
+}
+
+// Declare a variable called month with the number of the month.
+// Use an if statement to print the month's name.
+
+const month = 5;
+
+if (month === 1) {
+    console.log('January');
+} else if (month === 2) {
+    console.log('February');
+} else if (month === 3) {
+    console.log('March');
+} else if (month === 4) {
+    console.log('April');
+} else if (month === 5) {
+    console.log('May');
+} else if (month === 6) {
+    console.log('June');
+} else if (month === 7) {
+    console.log('July');
+} else if (month === 8) {
+    console.log('August');
+} else if (month === 9) {
+    console.log('September');
+} else if (month === 10) {
+    console.log('October');
+} else if (month === 11) {
+    console.log('November');
+} else if (month === 12) {
+    console.log('December');
+}
+
+// Declare a variable and assign a number to it.
+// Print "Divisible by 3" if the variable is divisible by 3.
+// Print "Divisible by 5" if the variable is divisible by 5.
+// Print "Divisible by 3 and 5" if the variable is divisible both by 3 and by 5.
+
+const number = 15;
+
+if (number % 3 === 0 && number % 5 === 0) {
+    console.log('Divisible by 3 and 5');
+} else if (number % 5 === 0) {
+    console.log('Divisible by 5');
+} else if (number % 3 === 0) {
+    console.log('Divisible by 3');
+}
+
+// Create a variable named radius and assign a number to it.
+// Create a variable named circleArea that stores the area of the circle using the formula area = π * radius^2.
+// Use MDN to find out how to use the PI number in JS.
+
+const radius = 56;
+const circleArea = Math.PI * radius^2
```

Comment:
The `^` operator is not what you think it is
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_XOR